wee_slack.py

```diff
@@ -2971,6 +2973,8 @@ def subprocess_thread_message(message_json, eventrouter, team, channel, history_
 
             if parent_message.thread_channel and parent_message.thread_channel.active:
                 parent_message.thread_channel.buffer_prnt(message.sender, parent_message.thread_channel.render(message), message.ts, tag_nick=message.sender_plain)
+            elif message.ts > channel.last_read and parent_message.subscribed: #TODO: What is the difference between this and message_replied, at least for alerting on threads?
+                parent_message.notify_thread(action="subscribed", sender_id=message_json["user"]) #TODO: Update thread last read mark?
```

Comment:
last read shouldn't be updated here as this will run when the message is received, and we shouldn't update last read until you read the message.

Reply:
Yep, this is just a reminder that it might be a good idea to update the thread last mark (wherever that should be done).